docs/servo_power_test/src/power_test_controller.ts

```diff
@@ -61,19 +54,27 @@ export class PowerTestController {
     await this.dutController.dut.close();
   }
   public async startMeasurement() {
-    if (this.ui.runnerNumber === 0) return;
+    if (this.ui.runnerNumber === 0) {
+      this.ui.setErrorMessage('No configuration/workload found.');
```

Comment:
Could you add steps to fix the error state as well? e.g. adding "Please click 'add config' button to setup an workload" or something.